<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CsvInput</class>
 <widget class="QDialog" name="CsvInput">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1418</width>
    <height>1078</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>421</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>InputConfiguration</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>1190</x>
        <y>910</y>
        <width>158</width>
        <height>77</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_loadColumn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>550</y>
        <width>381</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Load Columns</string>
      </property>
      <widget class="QListWidget" name="textControl_list">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>256</width>
         <height>181</height>
        </rect>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Entry A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Entry B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LENGHT</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btn_loadCols">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>491</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>Configuration settings</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>201</width>
         <height>178</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLineEdit" name="ed_startLine">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>133</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>StartLine</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="ed_endLine">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>EndLine</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Separator</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="ed_decimal">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Point(.)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Decimalpoint</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="ed_language">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>EN</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Language</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLineEdit" name="ed_Spacing">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>10.5</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Spacing</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="ed_Units">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Microns</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Units</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>290</y>
         <width>80</width>
         <height>83</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btn_CustomFormat">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>CustomFormat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_LoadConfig">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>LoadFormat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_CustomFormat_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>ExportXML</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_CoordCols">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>221</width>
         <height>191</height>
        </rect>
       </property>
       <property name="title">
        <string>Coordinates Colunm</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>196</width>
          <height>175</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLineEdit" name="ed_XStartCol">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_15">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Start_Posx</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="ed_yStart_Col">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_16">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Start_Posy</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="ed_z_StartCol">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_17">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Start_Posz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLineEdit" name="ed_X_EndCol">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="lbl_EndX_Col">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>End_Posx</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="ed_yEnd_Col_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_End_y_Col">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>End_Posy</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="ed_z_EndCol">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_20">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>End_Posz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_loadCSV">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>LoadFile</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>701</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="myLayout"/>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>91</width>
        <height>87</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>CSV-Data format</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="cmb_box_FileFormat">
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VolumeGraphics</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MAVI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>open_iA</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="cb_fmtEnglish">
         <property name="text">
          <string>ENGLISCH_FORMAT</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>add ID column</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>95</width>
        <height>92</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Column-Separator</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="cmb_box_separator">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>COMMA (,)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COLUMN (;)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Data Input-Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QComboBox" name="comboBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Fiber</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pores</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CsvInput</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CsvInput</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
